importer.py

```diff
@@ -239,32 +246,44 @@ def convert_relationships_to_comments(self, issue):
         relates_to = issue['is-related-to']
         depends_on = issue['depends-on']
         blocks = issue['blocks']
+        try:
+            epic_link = issue['epic-link']
+        except (AttributeError, KeyError):
+            epic_link = None
 
         for duplicate_item in duplicates:
             issue['comments'].append(
-                {"body": "<i>[Duplicates: " + self._replace_jira_with_github_id(duplicate_item)})
+                {"body": f'<i>[Duplicates: <a href="https://github.com/{self.options.account}/{self.options.repo}/issues?q=' + self._replace_jira_with_github_id(duplicate_item) + '">' + self._replace_jira_with_github_id(duplicate_item) + '</a>]</i>'})

```

Comment:
```suggestion
                {"body": f'<i>[Duplicates: <a href="https://github.com/{self.options.account}/{self.options.repo}/issues?q=%22' + self._replace_jira_with_github_id(duplicate_item) + '%22">' + self._replace_jira_with_github_id(duplicate_item) + '</a>]</i>'})

```

We should add double quotes for a strict GitHub search. I missed that for INFRA issues.

Ex:
- [no quote](https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+INFRA-2000) (wrongly include INFRA-325 which contains the text `2000`): `https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+INFRA-2000`
- [simple quotes](https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+%27INFRA-2000%27) (same): `https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+%27INFRA-2000%27`
- [double quotes](https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+%22INFRA-2000%22): `https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+%22INFRA-2000%22`

Reply:
doesn't seem to work, the last one still includes 2

what we want is this:
`in:title INFRA-2000`

https://github.com/jenkins-infra/helpdesk/issues?q=is%3Aissue+in%3Atitle+INFRA-2000

Reply:
I like the fact issue mentioning the searched one are included by default, as the searched one is still easily identifiable in the returned list, and seing the other can help (I know it does help me on helpdesk and "old" issues)

Reply:
We could also add a second link "(in title only)", WDYT?

Reply:
It's not catching mentioned ones in your example unless i'm missing something it's just matching `INFRA(NUMBER)-2000`

I think people can easily modify the query to find others but the link as-is is for finding the matching issue?

Reply:
This isn't a good example as the migration issue is catched in a lot of searches, I'll find better one(s) tomorrow.